### PR TITLE
Virtuoso 6.1.6 formula

### PR DESCRIPTION
**Pull request by icaromedeiros**
Virtuoso Open Source 6.1.6 version

### CODE REVIEW

**Comment by DomT4** [comment]
Hi! If this is still useful please address any comments ` brew audit --strict ` has locally, Thanks!

File: virtuoso616.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Virtuoso616 < Formula
+  homepage "http://virtuoso.openlinksw.com/wiki/main/"
+  url "https://downloads.sourceforge.net/project/virtuoso/virtuoso/6.1.6/virtuoso-opensource-6.1.6.tar.gz"
+  sha1 "03bc14b1627d16d76687f8b8659801966aab3fb4"
+
+  head do
```
Comment (DomT4) [inline virtuoso616.rb]: :scissors: this whole block. Versions formula can't have ` head ` blocks.

File: virtuoso616.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    url "https://github.com/openlink/virtuoso-opensource.git", :branch => "develop/6"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  # If gawk isn't found, make fails deep into the process.
+  depends_on "gawk" => :build
```
Comment (DomT4) [inline virtuoso616.rb]: Is this just a build-time dep? No runtime linkage?

**Comment by DomT4** [comment]
Closing per inactivity. Please feel free to request reopening or open a new PR if you're still interested - If you do, please fix any comments ` brew audit --strict ` has, and bump to the latest version of this branch if appropriate. Thanks!

File: virtuoso616.rb (hunk 2 above)
Comment (icaromedeiros) [inline virtuoso616.rb]: Yes, the same way Virtuoso7

https://github.com/Homebrew/homebrew/blob/fad929edd068dc3db40b38fa89ed215c6eb3887f/Library/Formula/virtuoso.rb

Comment (DomT4) [inline virtuoso616.rb]: Cool. No worries, Thanks for double checking!

**Pull request closed without merging**